src/store/index.ts

```diff
@@ -0,0 +1,28 @@
+import { configureStore } from '@reduxjs/toolkit'
+
+export default configureStore({
+    reducer: {
+        list: (state = {todos: []}, action) => {
+            switch (action.type) {
+                case 'ADD_TODO': {
+                    const newState = state;
+                    newState.todos.push(action.payload);
+                    return newState;
+                }
+                case 'REMOVE_TODO': {
```

Comment:
extract the reducer function definition "outside" instead of providing it inline, and also create separate definition for `state`:
```
interface TodosState { 
  todos: Todo[];
}
const todoReducer = (state, action) => {
  switch (action.type) {
                case 'ADD_TODO': {
                    const newState = state;
                    newState.todos.push(action.payload);
                    return newState;
                }
                case 'REMOVE_TODO': { ... }
}
```